README.md

```diff
@@ -1 +1,7 @@
-# microsoft-teams-test-tab
\ No newline at end of file
+# microsoft-teams-test-tab
+
+#How to deploy test app for sdk changes that are not release as npm package
+1. use yarn link to your microsoft-teams-library-js and do changes in app.ts
+2. run 'yarn build-<alias>' inorder to put outfile inside public/<alias> folder
```

Comment:
output

Reply:
@ydogandjiev , considering this PR safe i am going to complete this PR as i need these changes. I can work on your feedback in separate PR if any.